stockticker/stockticker.py

```diff
@@ -32,16 +33,75 @@ def query_quandl(ticker, quandl_key, value='open', days=500):
                       columns=[col['name'] for col in raw['datatable']['columns']])
     return df[['date', value]].rename({'date': 'ds', value: 'y'}, axis='columns')
 
+def query_prophet(prophet_url, secret_key, dates, prices):
+    """Makes a query to Prophet"""
+    prophet_params = {'ds': dates, 'y': prices, 'key': secret_key}
+
+    return requests.post(prophet_url, data=json.dumps(prophet_params))
+
+def get_error_div(error_string):
+    """Returns a div displaying an error message"""
+    return "<div>" + error_string + "</div>"
+
+def get_initial_js(ticker_handler_url, portfolio_handler_url,
```

Comment:
this function can be done within flask templating, I think it will be somewhat cleaner to do it that way.